Packs/Base/Scripts/DBotTrainClustering/DBotTrainClustering.py

```diff
@@ -637,46 +631,48 @@ def create_summary(model_processed: Type[PostProcessing], fields_for_clustering:
-        'Percentage of clusterized samples after selection (after Phase 1 and Phase 2)': "%s  (%s/%s)"
-                                                                                         % (
-                                                                                             str(percentage_selected_samples),
-                                                                                             str(nb_clusterized_after_selection),
-                                                                                             str(number_of_sample)),
-        'Percentage of clusterized samples (after Phase 1)': "%s  (%s/%s)" %
-                                                             (str(percentage_clusterized_samples),
-                                                              str(number_of_clusterized),
-                                                              str(number_of_sample)),
+        'Percentage of clusterized samples after selection (after Phase 1 and Phase 2)': "{}  ({}/{})".format(
+            str(percentage_selected_samples),
+            str(nb_clusterized_after_selection),
+            str(number_of_sample)),
+        'Percentage of clusterized samples (after Phase 1)': "{}  ({}/{})".format(str(percentage_clusterized_samples),
+                                                                                  str(number_of_clusterized),
+                                                                                  str(number_of_sample)),
         'Percentage of cluster selected (Number of high quality groups/Total number of groups)':
-            "%s  (%s/%s)" %
-            (str(percentage_clusters_selected),
-             str(number_clusters_selected),
-             str(nb_clusters)),
+            f"{percentage_clusters_selected}  ({number_clusters_selected}/{nb_clusters})",
         'Fields used for training': ' , '.join(fields_for_clustering),
         'Fields used for cluster name': field_for_cluster_name[0] if field_for_cluster_name else "",
         'Training time': str(model_processed.date_training)
     }
     return summary
 
 
-def return_entry_clustering(output_clustering: Dict, tag: str = None) -> None:
+def return_entry_clustering(output_clustering: str, tag: str = None) -> None:
     """
     Create and return entry with the JSON containing the clusters
     :param output_clustering: json with the cluster
     :param tag: tag
     :return: Return entry to demisto
     """
+    outputs = [
+        {
+            'name': clst['name'],
+            'incidents_ids': clst['incidents_ids']
+        }
+        for clst in json.loads(output_clustering)['data']
+    ]
     return_entry = {
         "Type": entryTypes["note"],
         "ContentsFormat": formats['json'],
         "Contents": output_clustering,
-        "EntryContext": {'DBotTrainClustering': output_clustering},
+        "EntryContext": {'DBotTrainClustering': outputs},
```

Comment:
does this break the context outputs? if yes, lets add all outputs now to the yml and release with breaking changes. 
Do we need to have breaking changes?